plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/convert/AppEngineStandardProjectConvertJob.java

```diff
@@ -48,14 +43,7 @@ protected IStatus run(IProgressMonitor monitor) {
 
     // Updating project before installing App Engine facet to avoid
     // https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/1155.
-    try {
-      GpeMigrator.removeObsoleteGpeRemnants(facetedProject, subMonitor.newChild(20));
-    } catch (CoreException ex) {
-      // Failed to remove GPE remnants; live with it.
-      subMonitor.setWorkRemaining(20);
-      subMonitor.worked(20);
-      logger.log(Level.WARNING, "Error while removing GPE remnants", ex);
```

Comment:
We need to catch this and don't error out, because it will always throws the exception, e.g., if the project is not a GPE project (e.g., a plain Java project).

Because we swallow the exception, the logger is for us to get notified if the conversion fails for whatever reason.